phpunit

```diff
@@ -10,12 +10,10 @@ if (!file_exists(__DIR__.'/vendor/symfony/phpunit-bridge/bin/simple-phpunit')) {
     exit(1);
 }
 if (!getenv('SYMFONY_PHPUNIT_VERSION')) {
-    if (\PHP_VERSION_ID < 70200) {
-        putenv('SYMFONY_PHPUNIT_VERSION=7.5');
```

Comment:
This was a leftover from Symfony 4 wasn't it?

Reply:
I'd keep this, the bridge is still declared as compatible with <7.2